docs/DEPLOYMENT.md

```diff
@@ -171,21 +206,31 @@ metadata:
   name: ui
   namespace: tekton-hub
 data:
-  API_URL: API URL   <<< update this by api routes
+  API_URL: API URL   <<< update this by api route
```

Comment:
Although it is very obvious to get the api route using `oc get route` command. But to be on a safer side can we also add this command here as an comment?